Add media type (#944)
radiantearth/stac-spec: catalog-spec/catalog-spec.md

**Note:** A link to at least one `item` or `child` catalog is **REQUIRED**.

## Media Type for STAC Catalogs

A STAC Catalog is a JSON file ([RFC 8259](https://tools.ietf.org/html/rfc8259)), and thus should use the 
`[application/json](https://tools.ietf.org/html/rfc8259#section-11)` as the [Media Type](https://en.wikipedia.org/wiki/Media_type) 

> (previously known as the MIME Type). STAC Catalogs are recommended to additional specify a `profile` equal to `stac-catalog` to

Reviewer: In media type terminology `profile` is a parameter. Applies to all three types.

```suggestion
(previously known as the MIME Type). STAC Catalogs are recommended to additionally add the parameter `profile` with the value `stac-catalog` to
```
